jetstream/config.py

```diff
@@ -478,18 +489,23 @@ def from_dict(cls, d: dict) -> "MetricsSpec":
             for k, v in d.items()
             if k not in known_keys and k != "28_day"
         }
+
+        params["parameters"] = d.get("parameters", dict())
+
         return cls(**params)
 
     def resolve(
-        self, spec: "AnalysisSpec", experiment: ExperimentConfiguration
+        self,
+        spec: "AnalysisSpec",
+        experiment: ExperimentConfiguration,
+        config_parameters: Optional[Dict[Any, Any]] = dict(),
```

Comment:
I think we should be consistent with naming
```suggestion
       parameters: Optional[Dict[Any, Any]] = dict(),
```